src/promptflow-evals/promptflow/evals/evaluate/_utils.py

```diff
@@ -64,66 +70,106 @@ def _write_properties_to_run_history(properties: dict) -> None:
         LOGGER.error("Fail writing properties '%s' to run history: %s", properties, e)
 
 
-def _azure_pf_client_and_triad(trace_destination):
-    from promptflow.azure._cli._utils import _get_azure_pf_client
+def _get_credential(force_cli: bool = False) -> TokenCredential:

```

Comment:
We are only getting credentials here and we not use the pf-azure run in this code, so that we can avoid overhead, for creation of pfromptflow-azure client.

Reply:
I don't see the need for _get_credential and _get_ml_client. Can we just reuse get_client_for_cli from promptflow-azure?

Reply:
Removed.